ui/src/components/Dashboard/index.jsx

```diff
@@ -246,32 +246,36 @@ export default class Dashboard extends Component {
       ...props
     } = this.props;
     const { error, navOpen, showHelpView, deploymentVersion } = this.state;
-    const drawer = (
-      <nav className={classes.nav}>
-        <div>
-          <div
+    const logoWithApplicationName = (
+      <>
```

Comment:
Replace `<>` with `<Fragment>` to stay consistent with the rest of the codebase.

Reply:
Oops, yeah definitely!